metadata.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "puppetlabs-nodejs",
+  "name": "puppetcommunity-nodejs",
   "version": "0.8.0",
   "author": "puppetlabs",
   "summary": "Install Node.js package and npm package provider.",
   "license": "Apache-2.0",
-  "source": "https://github.com/puppetlabs/puppetlabs-nodejs",
-  "project_page": "https://github.com/puppetlabs/puppetlabs-nodejs",
+  "source": "https://github.com/puppet-community/puppet-nodejs",
+  "project_page": "https://github.com/puppet-community/puppet-nodejs",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
```

Comment:
This is a bit uncertain. Should we continue to use puppetlabs MODULES for this or revert to using github issues?